modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-cluster-tasks-ref_{context}"]
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+
+`ClusterTask` functionality is deprecated since version 1.10 and will be removed in a future release. If your pipelines use `ClusterTasks`, you can re-create them using the tasks that are available from the {pipelines-shortnames} by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
+
+You cannot specify a custom execution image in any of the tasks available in the {pipelines-shortnames} installation namespace. These tasks do not support parameters such as `BUILDER_IMAGE`, `gitInitImage`, or `KN_IMAGE`. If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.
```

Comment:
> If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.

Is there a Procedure to reference here? Seems like a one-liner that may actually be more complex behind the scenes but also idk anything about this so could just be a misinterpretation.

Reply:
Fair point but out of scope. Created https://issues.redhat.com/browse/RHDEVDOCS-6087